saleor/graphql/webhook/mutations.py

```diff
@@ -359,14 +365,15 @@ def validate_input(cls, info, **data):
         query = data.get("query")
         object_id = data.get("object_id")
 
-        event_type = get_event_type_from_subscription(query) if query else None
-        if not event_type:
+        event_types = get_event_type_from_subscription(query) if query else None
+        if not event_types:
             raise_validation_error(
                 field="query",
                 message="Can't parse an event type from query.",
                 code=WebhookDryRunErrorCode.UNABLE_TO_PARSE,
             )
 
+        event_type = event_types[0] if event_types else None
```

Comment:
Here we assume that if there are multiple events we always take first? I would add a comment that we have this assumption.

Reply:
For `webhookDryRun` and `webhookTrigger` mutation there is already an info in description like: _Supports a single event (the first, if multiple provided)_

For `webhookCreate` and `webhookUpdate` we are able to parse and handle multiple events